### ISSUE TITLE
up/down arrows skip lines

### ISSUE THREAD

**Issue by deanmalmgren** [avendael/atomic-emacs#35]
I'm sure this is related to #34 but, for what its worth, from using the Keybinding Resolver (`cmd+.`) it looks like the `atomic-emacs:previous-line` and `atomic-emacs:next-line` end up skipping lines:

![key_bindings](https://cloud.githubusercontent.com/assets/255672/5524754/4669c12e-89a0-11e4-9f43-11948fc27499.png)

For what its worth, I have atom 0.165.0 and atomic-emacs 0.3.5.

As a workaround, I overrode this behavior in my keymap.cson with something like:

```cson
'.editor':
  'up': 'core:move-up'
  'down': 'core:move-down'
```

**Comment by LukeMauldin** [comment]
I am experiencing the same behavior.

**Comment by avendael** [comment]
Thanks for the inputs everyone, it will help me debug this issue.

**Comment by avendael** [comment]
Fixed by d92c41a748e5f12c727fc472d0e34e557c7a2693. Apparently, those lines aren't needed anymore. Please update to `0.3.6` :)

**Comment by deanmalmgren** [comment]
thanks for the update! things appear to be working properly now.
